packages/palette/src/elements/Input/Input.tsx

```diff
@@ -143,7 +143,7 @@ const StyledInput = styled.input<StyledInputProps>`
   padding: 0 ${themeGet("space.1")};
   appearance: none;
   line-height: 1;
-  border: 1px solid ${themeGet("colors.black15")};
```

Comment:
**question:** Is this change intentional?

Reply:
Yes actually, since we don't have `black15` in our design system anymore :)